amcl/src/amcl_node.cpp

```diff
@@ -802,6 +802,11 @@ AmclNode::handleMapMessage(const nav_msgs::OccupancyGrid& msg)
            msg.info.width,
            msg.info.height,
            msg.info.resolution);
+  
+  if(msg.header.frame_id != global_frame_id_)
+  		ROS_WARN("Frame_id of map received:'%s' doesn't match global_frame_id:'%s;'. This could cause issues with reading published topics",
```

Comment:
this PR has a mix of tabs + spaces -- which is why github is rendering it odd. The amcl code base isn't perfect on style guide, but I do try to ask new code be mostly compatible. Could you update your changes to use ONLY spaces, and in particular 2 spaces indent for each new block (you can just push a new commit and I will squash later). Thanks!